crates/db_schema/src/impls/comment_reply.rs

```diff
@@ -65,8 +65,21 @@ impl CommentReply {
     for_comment_id: CommentId,
   ) -> Result<Self, Error> {
     let conn = &mut get_conn(pool).await?;
-    comment_reply
-      .filter(comment_id.eq(for_comment_id))
+    comment_reply::table
+      .filter(comment_reply::comment_id.eq(for_comment_id))
+      .first::<Self>(conn)
+      .await
+  }
+
+  pub async fn read_by_comment_and_person(
+    pool: &mut DbPool<'_>,
+    for_comment_id: CommentId,
+    for_recipient_id: PersonId,
+  ) -> Result<Self, Error> {
+    let conn = &mut get_conn(pool).await?;
+    comment_reply::table
+      .filter(comment_reply::comment_id.eq(for_comment_id))
+      .filter(comment_reply::recipient_id.eq(for_recipient_id))
```

Comment:
Could do an and in the filter instead of 2 separate filters.

Reply:
Either works, but most of the code uses multiple filters like this.

Reply:
Makes sense. I'm curious if there is a noticeable performance difference between the 2.